### PR TITLE
[WIP] README and CONTRIBUTING files

### PR DESCRIPTION
**Pull request by Bidjii**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,36 @@
...
 
-## CLI
-There is an internal set of commands available, to bkirk. To bootstrap our new componentst:
+## Installation
+BlaBlaCar `ui-library` is available as an [npm package](https://www.npmjs.com/package/@blablacar/ui-library)
 
+Just run this inside your project
 ```
-npm run kirk
+npm i @blablacar/ui-library
```
Comment (pith) [inline README.md]: Maybe you should also add yarn to seem fancy

File: README.md [hunk 2]
```diff
@@ -1,8 +1,36 @@
...
+But again, don't forget that you need to have React to make it work.
+
+## Changelog
+Check the [changelog](https://github.com/blablacar/ui-library/blob/master/CHANGELOG.md) to stay informed on all updates
+
+## Examples and documentation
+You can find examples and tech documentation [here](https://blablacar.github.io/ui-library/).
+
+## Contributing
+Wanna contribute ? [This way.](https://github.com/blablacar/ui-library/blob/master/LICENSE) :)
```
Comment (pith) [inline README.md]: Wrong link

File: README.md [hunk 3]
```diff
@@ -1,8 +1,36 @@
...
+## Installation
+BlaBlaCar `ui-library` is available as an [npm package](https://www.npmjs.com/package/@blablacar/ui-library)
 
+Just run this inside your project
 ```
-npm run kirk
+npm i @blablacar/ui-library
 ```
+
+## Usage
```
Comment (pith) [inline README.md]: You should also document how to use it for server side rendering with styled-jsx

Comment (pith) [reply README.md]:
```
import { flushToHTML } from '@blablacar/ui-library'

const index = `
    <!doctype html>
    <html>
      <head>
        ${title}
        ${flushToHTML()}
      </head>
    ...
    `
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,31 +1,35 @@
...
-- A unit test file.
-- A story file. (functional testing)
+Each component is stored in the `src` folder. Each component has its own folder with all of its information. Basically, you will always have:
+- A component `.tsx` file
+- A stylesheet associated to your component
+- A unit test file
+- A story file (functional testing)
 
-The only dependency to use this library is to be in a ReactJS environment. The repository also includes ES6+Typescript syntax but this one is compiled to ES5 in the build folder.
+The only dependency to use this library is to be in a ReactJS environment. The repository also includes ES6 + Typescript syntax but this one is compiled to ES5 in the build folder.
```
Comment (loick) [inline CONTRIBUTING.md]: remove ES6

File: README.md (hunk 3 above)
Comment (loick) [reply README.md]: Agreed!

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -38,8 +42,8 @@ The package is composed by several precommit hooks:
 # Git
 
 To automatically prepend the branch name to your commit name on every commit, please run the following command at the root of the repository:
-`cp ./docs/setup/prepare-commit-msg .git/hooks/prepare-commit-msg`
+`cp ./bin/setup/prepare-commit-msg .git/hooks/prepare-commit-msg`
 
 # Editor Setup
 
-We recommend you to add to your text-editor some plugins such as `tslint` or `stylelint` to immediately get the errors without waiting for the precommit-hook crash.
+We recommend you to add to your text-editor some plugins such as `tslint` or `stylelint` to immediately get the errors without waiting for the precommit-hook crash.
```
Comment (pith) [inline CONTRIBUTING.md]: do we use prettier on kirk ?

File: README.md (hunk 3 above)
Comment (Bidjii) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (Bidjii) [reply README.md]: :trollface:

File: CONTRIBUTING.md (hunk 5 above)
Comment (Bidjii) [reply CONTRIBUTING.md]: Nope, not at the moment

File: README.md [hunk 6]
```diff
@@ -1,8 +1,54 @@
...
+import { Button, Input } from '@blablacar/ui-library'
+```
+But again, don't forget that you need to have React to make it work.
+
+
+For server-rendering purposes, just use the `flushToHTML` from styled-jsx that we export from the `ui-library`
+```javascript
+import { flushToHTML } from '@blablacar/ui-library'
+
+const index = `
```
Comment (pith) [inline README.md]: Maybe the example of styled-jsx is better 😬 

```
import React from 'react'
import ReactDOM from 'react-dom/server'
import { flushToHTML } from '@blablacar/ui-library'
import App from './app'

export default (req, res) => {
  const app = ReactDOM.renderToString(<App />)
  const styles = flushToHTML()
  const html = `<!doctype html>
    <html>
      <head>${flushToHTML()}</head>
      <body>
        <div id="root">${app}</div>
      </body>
    </html>`
  res.end(html)
}
```

**Pull request merged**